### DISCUSSION TITLE
001

### DISCUSSION THREAD

**Discussion by giscus** [zhiyu1998/rrorange-and-friends-weekly#2] [General]
# 001

《大橘和朋友们的周刊》： 分享日常冲浪互联网看到好玩的网站、app应用、资源分享、效率软件工具集等

https://zhiyu1998.github.io/rrorange-and-friends-weekly/posts/2024/001

**Comment by loveliveao** [comment]
# 大橘，给你点赞！

**Comment by xusi4git** [comment]
有几个软件都挺好用的，谢谢分享
